distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -1264,3 +1274,8 @@ slice_text text char_ranges =
     char_ranges.map char_range->
        sb.append text char_range.start char_range.end
     sb.toString
+
+Span.from (that:Utf_16_Span) = that.to_grapheme_span
+
+Utf_16_Span.from (that:Span) = that.to_utf_16_span
```

Comment:
I'm wondering if from-conversions should be treated in the same way as extension methods import-wise.

It feels wrong to me to place them here rather then next to their relevant types (`Span`).

Actually my intuition would be that if the from-conversion is defined in the same file as the corresponding type, it should be 'always' registered - i.e. regardless how we got an instance of that type in scope, the `from` method should find this conversion.

cc: @hubertp what do you think? Also not sure if this is even possible.

Reply:
I think how we use `from` conversions needs to mature. For now I put these two in a reasonably close space.

I also don't think they can be member functions (though could be wrong on that) so this seemed a reasonable workaround for now.

Reply:
As these are not currently used will remove these ones for now.